### PR TITLE
feat: add next queries

### PR DESCRIPTION
**Pull request by diegopf**
[EX-6306](https://searchbroker.atlassian.net/browse/EX-6306)

### CODE REVIEW

**Review by mnavarroespinosa — COMMENTED** [review]
Could you move the folder `src/schemas/responses/models` up to `src/schemas/models`? We previously agreed to have it at the same level as `responses` and `requests`

**Comment by mnavarroespinosa** [comment]
Could you add the exports for `/models`, `/requests`, and `/responses` to `schemas/index.ts`? And also to `types/index.ts`?

File: packages/x-adapter-platform/src/types/requests/next-queries-request.model.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export interface BasePlatformNextQueriesRequest extends PlatformQueryableRequest {}
+
+/**
+ * Request for the `next queries` endpoint with `extra params`.
+ *
+ * @public
+ */
+export interface PlatformNextQueriesRequest
+  extends BasePlatformNextQueriesRequest,
+    PlatformExtraParamsRequest {}
```
Comment (tiborux) [inline packages/x-adapter-platform/src/types/requests/next-queries-request.model.ts:17]:
```suggestion
export interface PlatformNextQueriesRequest
  extends BasePlatformNextQueriesRequest,
  PlatformQueryableRequest,
    PlatformExtraParamsRequest {}
```

File: packages/x-adapter-platform/src/types/requests/next-queries-request.model.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { PlatformExtraParamsRequest, PlatformQueryableRequest } from './request.types';
+
+/**
+ * Base request for the `next queries` endpoint.
+ *
+ * @public
+ */
+export interface BasePlatformNextQueriesRequest extends PlatformQueryableRequest {}
```
Comment (tiborux) [inline packages/x-adapter-platform/src/types/requests/next-queries-request.model.ts:8]: Remove this type

File: packages/x-adapter-platform/src/types/requests/request.types.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/**
```
Comment (tiborux) [inline packages/x-adapter-platform/src/types/requests/request.types.ts:1]: Export just the needed ones:

  PlatformQueryableRequest, PlatformExtraParamsRequest

**Review by tiborux — DISMISSED** [review]
gj

**Comment by mnavarroespinosa** [comment]
gj!

**Review by mnavarroespinosa — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Review by mnavarroespinosa — APPROVED** [review]

**Pull request merged**